corehq/apps/cloudcare/templates/too_many_ucrs.html

```diff
@@ -0,0 +1,13 @@
+{% extends "error_base.html" %}
+{% load hq_shared_tags %}
+{% load i18n %}
+{% block title %}
+  {% trans "Blocked" %}
+{% endblock %}
+
+{% block page_name %}{% trans "You are currently blocked from using Web Apps" %}{% endblock %}
+{% block page_content %}
+<div>
```

Comment:
I would remove the unstyled `div` here and add `class="error"` to the `<p>` tag

Reply:
a1f41126477e116b471718cc09c8149a07f107d6